src/Entity/Activity/Activity.php

```diff
@@ -438,4 +443,14 @@ public function setPresent(?int $present)
     {
         $this->present = $present;
     }
+
```

Comment:
again, naming stuff.... getVisible/setVisible feels more natural

Reply:
In this case, I feel that Hidden is better, as it might return null, which should lead to the same behaviour as false. With Visible, null should perform similar to true. However, this would lead to weird constructs instead of checking for a "falsy" value.